deploy/gcp/README.md

```diff
@@ -80,6 +85,12 @@ $ kubectl create serviceaccount -n kube-system tiller
 serviceaccount/tiller created
 $ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
 $ helm init --service-account tiller --wait
+=======
+```sh
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
+>>>>>>> use more azure cli commands
```

Comment:
Remove merge conflict here, and please keep the original Helm2 install instructions in this Helm2 section